CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v1.3.0]
+### Changed
+Added OpenShift 4 support
```

Comment:
You can add this to v1.2.0, as long as you haven't tagged that version, you don't need to make a new minor version for each change.